simplemma/__init__.py

```diff
@@ -7,8 +7,9 @@
 __version__ = "0.9.1"
 
 
-from .language_detector import in_target_language, lang_detector
+from .language_detector import in_target_language, langdetect
 from .lemmatizer import lemmatize, lemma_iterator, text_lemmatizer, is_known
 from .tokenizer import simple_tokenizer
+from .token_sampler import TokenSampler
```

Comment:
I'm not sure why token sampling should be performed by a separate module, it's only used for language detection isn't it?

Reply:
Yup. But it's a separate class.

Users can extend TokenSampler and provide their own sampling mechanisms.
Also, in the future, this library could add other functionality using sampling.

I just tried to keep the 1 file = 1 class approach.

Reply:
OK.